src/RepoAutomation.Core/Models/BranchProtectionPolicy.cs

```diff
@@ -6,13 +6,35 @@ public class BranchProtectionPolicy : BaseModel
     public bool strict { get; set; }
     public Enforce? enforce_admins { get; set; }
     public Enforce? required_conversation_resolution { get; set; }
+    public bool? required_pull_request_reviews { get; set; }
+    public bool? restrictions { get; set; }
+}
+
+//Different structure when updating in PUT
+public class BranchProtectionPolicyPut : BaseModel
```

Comment:
It seems like the Put action has a different definition - so I use a different object for Puts for now.